Expensify/App
# Fix signin page layout styles

### Details
There was an issue with rendering on Desktop Safari/iOS Web for any pages that used the sign in page styles where they wouldn't flex to cover the entire document body. This is due to how Apple's Webkit renders flex styles slightly differently from how Chrome/other browsers do.

I added `flex: 1` to the sign-in page styles to fix this.

### Fixed Issues
Expensify/Expensify#157816

### Tests
1. Navigate to the sign in page, verify it renders correctly (see screenshot).
2. Navigate to the set password page, verify it renders correctly (see screenshot).


### Tested On

### Screenshots

#### Web
<img width="1790" alt="Screen Shot 2021-03-19 at 12 23 38 PM" src="https://user-images.githubusercontent.com/31285285/111731245-0553d800-88ae-11eb-9552-f4c7daacf902.png">
<img width="1787" alt="Screen Shot 2021-03-23 at 8 02 28 PM" src="https://user-images.githubusercontent.com/31285285/112143354-b93cc680-8c12-11eb-97b0-916962e1d48a.png">

#### Mobile Web
<img width="430" alt="Screen Shot 2021-03-19 at 12 24 00 PM" src="https://user-images.githubusercontent.com/31285285/111731261-0ab12280-88ae-11eb-8cf7-78eab0437877.png">
<img width="375" alt="Screen Shot 2021-03-22 at 12 10 50 PM" src="https://user-images.githubusercontent.com/31285285/111939627-b3a68a00-8b07-11eb-982d-7d9f32085d20.png">
